### PR TITLE
ES6 migrate 'view_model/bindings'

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/view_model/bindings/ConversationListBindings.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        if (($element.height() <= 0) || !$element.is_scrollable()) {
+          return archive_column.removeClass('left-list-center-border-bottom conversations-center-border-top');
+        }
+
+        archive_column.toggleClass('left-list-center-border-top', !$element.is_scrolled_top());
+        archive_column.toggleClass('left-list-center-border-bottom', !$element.is_scrolled_bottom());
+      });
+    }
+  }
+  , 100);
```
Comment (lipis) [inline app/script/view_model/bindings/ConversationListBindings.js:37]: Magic :)

File: app/script/view_model/bindings/MessageListBindings.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+          .then(function(blob) {
+            $(element).removeClass('image-loading');
+            object_url = window.URL.createObjectURL(blob);
+            image_element[0].src = object_url;
+            viewport_subscription.dispose();
+          })
+          .catch(() => {});
+      }
+    }
+    , 500);
```
Comment (lipis) [inline app/script/view_model/bindings/MessageListBindings.js]: moar

File: app/script/view_model/bindings/ConversationListBindings.js (hunk 1 above)
Comment (gregor) [reply app/script/view_model/bindings/ConversationListBindings.js:37]: Feel free to add the rule and clean it up. A simple timeout is not really that magic. I am es6 migrating, not rewriting all of our code.

Comment (lipis) [reply app/script/view_model/bindings/ConversationListBindings.js:37]: _kill two birds with one stone_ :)
giphy-downsized-large

**Comment by lipis** [comment]
We should go less than 10% with this one :)

**Pull request merged**